src/Typography/Typography.styles.ts

```diff
@@ -0,0 +1,34 @@
+import styled, { css } from 'styled-components';
+
+import BaseSpacing from '../BaseSpacing';
+import { getColorFromTheme } from '../shared/mixins';
+import { TypographyProps } from './types';
+import { STYLES } from './constants';
+
+export const BaseTypography = styled(BaseSpacing)<TypographyProps>`
+  ${({ theme }) => css`
+    font-family: ${theme.typography.font.primary};
+  `}
+
+  ${({ variant = 'body1' }) => STYLES[variant]}
+
+  ${({ color, theme }) =>
+    color &&
+    css`
+      color: ${getColorFromTheme(theme, color)};
+    `}
+
+  ${({ weight }) =>
```

Comment:
Would you rename the prop `weight` to `fontWeight` for style attributes naming consistency?

Reply:
Changed